### PR TITLE
PMM-7999 Add QuickStart section

### PR DESCRIPTION
**Pull request by rasika-chivate**
1. Add a QuickStart section to the PMM doc
2. Restructure the Toc to fit in the Quickstart section

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+    ```sh
+    curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+    ```
+
+=== "wget"
+
+    ```sh
+    wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+    ```
```
Comment (fiowro) [inline docs/quickstart.md:36]:
```suggestion
    === "cURL"

        ```sh
        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
        ```

    === "wget"

        ```sh
        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
        ```
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Quickstart Guide
+
+
+This guide covers how you can quickly get started using PMM. This is applicable to Docker compatible Linux based systems.
```
Comment (catalinaadam) [inline docs/quickstart.md:4]:
```suggestion
This guide helps you quickly get started with PMM using Docker-compatible Linux-based systems.
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+## Prerequisites
+
+[Docker](https://docs.docker.com/engine/install/) 1.12.6 or higher.
+
+## System requirements
+
+**Disk**
+
+Approximately 1 GB of storage per monitored database node with data retention set to one week.
```
Comment (catalinaadam) [inline docs/quickstart.md:15]:
```suggestion
Approximately 1 GB of storage per monitored database node, with data retention set to one week. 
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Approximately 1 GB of storage per monitored database node with data retention set to one week. 
+
+**Memory**
+
+A minimum of 2 GB per monitored database node. The increase in memory usage is not proportional to the number of nodes. For example, data from 20 nodes should be easily handled with 16 GB.
+
+**Ports**
+
+This is a list of [ports](setting-up/server/network.md) used by the various components of PMM.
```
Comment (catalinaadam) [inline docs/quickstart.md:23]:
```suggestion
Check out the [list of ports used by PMM components](setting-up/server/network.md).
```

File: docs/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+For PMM to work correctly, your system’s firewall should allow TCP traffic on these ports (UDP is not needed).
+
+## Install PMM
+
+!!! caution alert alert-warning "Important"
+    - You can use the [easy installation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/easy-install.html) script that will verify and install any missing software and dependencies. To use it, run the command with sudo privileges or as root.
```
Comment (catalinaadam) [inline docs/quickstart.md:30]:
```suggestion
```

File: docs/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    - You can download the script from [github](https://github.com/percona/pmm/blob/main/get-pmm.sh).
```
Comment (catalinaadam) [inline docs/quickstart.md:31]:
```suggestion
```

File: docs/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+Here are the steps for installing the PMM server:
+
```
Comment (catalinaadam) [inline docs/quickstart.md:34]:
```suggestion
1. Run the [PMM easy installation script](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/easy-install.html) to verify and install any missing software and dependencies. The install script only runs on Linux-compatible systems. To use it, run the command with sudo privileges or as root.
```

File: docs/quickstart.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+For PMM to work correctly, your system’s firewall should allow TCP traffic on these ports (UDP is not needed).
+
+## Install PMM
+
+!!! caution alert alert-warning "Important"
+    - You can use the [easy installation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/easy-install.html) script that will verify and install any missing software and dependencies. To use it, run the command with sudo privileges or as root.
+    - You can download the script from [github](https://github.com/percona/pmm/blob/main/get-pmm.sh).
+
+Here are the steps for installing the PMM server:
```
Comment (catalinaadam) [inline docs/quickstart.md:33]:
```suggestion
To install PMM Server:
```

File: docs/quickstart.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+
+1. Set up Docker on [Mac](https://docs.docker.com/docker-for-mac/install) or [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu).
```
Comment (catalinaadam) [inline docs/quickstart.md:35]:
```suggestion
2. [Set up Docker on Mac](https://docs.docker.com/docker-for-mac/install) or [on Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu).
```

File: docs/quickstart.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+
+2. Install the PMM server using `cURL` or `wget` as follows:
```
Comment (catalinaadam) [inline docs/quickstart.md:37]:
```suggestion
2. Install the PMM Server using `cURL` or `wget`:
```

File: docs/quickstart.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+        ```
+
+    === "wget"
+
+        ```sh
+        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+        ```
+
+3. Log in to PMM with the default login credentials provided after the installation is completed.
```
Comment (catalinaadam) [inline docs/quickstart.md:52]:
```suggestion
3. Log into PMM with the default credentials provided after completing the installation.
```

File: docs/quickstart.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+
+For detailed instructions on installing the PMM server with the other methods, see [Setting up PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/index.html).
```
Comment (catalinaadam) [inline docs/quickstart.md:54]:
```suggestion
For instructions on installing the PMM Server with other methods, see [Setting up PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/index.html).
```

File: docs/quickstart.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+
+## Add a database instance
+
+=== "MySQL 5.7 or 8.0"
+
+    Before you add a MySQL database for monitoring you should have a [database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html#create-a-database-account-for-pmm).
```
Comment (catalinaadam) [inline docs/quickstart.md:60]:
```suggestion
   **Prerequisites**
   Before adding a MySQL database for monitoring,  [create a database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html#create-a-database-account-for-pmm).
```

File: docs/quickstart.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+
+    Here are the steps to add a MySQL database for monitoring:
```
Comment (catalinaadam) [inline docs/quickstart.md:62]:
```suggestion
    To add a MySQL database for monitoring:
```

File: docs/quickstart.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+
+    1. From the PMM UI, go to **Configuration** → **PMM Inventory** → **Add Instance**.
```
Comment (catalinaadam) [inline docs/quickstart.md:64]:
```suggestion
    1. From the PMM UI, go to **Configuration > PMM Inventory > Add Instance** and select **MySQL**.
```

File: docs/quickstart.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+
+    2. Select **MySQL**. The **Add Service** page opens.
```
Comment (catalinaadam) [inline docs/quickstart.md:66]:
```suggestion
```

File: docs/quickstart.md [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+
+    3. Enter your database credentials on the resulting page without changing any values.
```
Comment (catalinaadam) [inline docs/quickstart.md:68]:
```suggestion
    2. On the **Add Service** page, enter your database credentials without changing any values.
```

File: docs/quickstart.md [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Optional: Enter the information on the **Labels** and **Additional Options** section.
```
Comment (catalinaadam) [inline docs/quickstart.md:70]:
```suggestion
    4. (Optional) Enter the information on the **Labels** and **Additional Options** section. 
```

File: docs/quickstart.md [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Optional: Enter the information on the **Labels** and **Additional Options** section. 
+
+    5. Click **Add Service** at the bottom.
+
+    For detailed information, see [Adding a MySQL database for monitoring](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html).
+
+=== "MongoDB"
+
+    Before you add a MongoDB database for monitoring you should have a [database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html#create-pmm-account-and-set-permissions).
```
Comment (catalinaadam) [inline docs/quickstart.md:78]:
```suggestion
    Prerequisite:
    Before you add a MongoDB database for monitoring,  [create a database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html#create-pmm-account-and-set-permissions).
```

File: docs/quickstart.md [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+
+    Here are the steps to add a MongoDB database for monitoring:
```
Comment (catalinaadam) [inline docs/quickstart.md:80]:
```suggestion
    To add a MongoDB database for monitoring:
```

File: docs/quickstart.md [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+
+    1. From the PMM UI, go to **Configuration** → **PMM Inventory** → **Add Instance**.
```
Comment (catalinaadam) [inline docs/quickstart.md:82]:
```suggestion
    1. From the PMM UI, go to **Configuration >PMM Inventory > Add Instance** and select **MongoDB**.
```

File: docs/quickstart.md [hunk 22]
```diff
@@ -0,0 +1,157 @@
...
+
+    2. Select **MongoDB**. The *Add Service* page opens.
```
Comment (catalinaadam) [inline docs/quickstart.md:84]:
```suggestion
```

File: docs/quickstart.md [hunk 23]
```diff
@@ -0,0 +1,157 @@
...
+
+    3. Enter your database credentials on the resulting page without changing any values.
```
Comment (catalinaadam) [inline docs/quickstart.md:86]:
```suggestion
    3. On the  **Add Service** page, enter your database credentials without changing any values.
```

File: docs/quickstart.md [hunk 24]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Optional: Enter the information in the **Labels** and **Additional Options** section.
```
Comment (catalinaadam) [inline docs/quickstart.md:88]:
```suggestion
    4. (Optional) Enter the information in the **Labels** and **Additional Options** section. 
```

File: docs/quickstart.md [hunk 25]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Optional: Enter the information in the **Labels** and **Additional Options** section. 
+
+    5. Click **Add Service** at the bottom.
+
+    For detailed information on adding a MongoDB database, see [Adding a MySQL database for monitoring](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html).
+
+=== "PostgreSQL"
+
+    Before you add a PostgreSQL database for monitoring you should have a [database account for PMM]((https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#create-a-database-account-for-pmm).
```
Comment (catalinaadam) [inline docs/quickstart.md:96]:
```suggestion
    **Prerequisite**: Before adding a  PostgreSQL database for monitoring, [create a database account for PMM]((https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#create-a-database-account-for-pmm).
```

File: docs/quickstart.md [hunk 26]
```diff
@@ -0,0 +1,157 @@
...
+    
+    Here are the steps to add a PostgreSQL database for monitoring:
```
Comment (catalinaadam) [inline docs/quickstart.md:98]:
```suggestion
    To add a PostgreSQL database for monitoring:
```

File: docs/quickstart.md [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+
+    1. From the PMM UI, go to *Configuration* → *PMM Inventory* → *Add Instance*.
```
Comment (catalinaadam) [inline docs/quickstart.md:100]:
```suggestion
    1. From the PMM UI, go to **Configuration > PMM Inventory > Add Instance** and choose **PostgreSQL**.
```

File: docs/quickstart.md [hunk 28]
```diff
@@ -0,0 +1,157 @@
...
+
+    2. Select **PostgreSQL**. The **Add Service** page opens.
```
Comment (catalinaadam) [inline docs/quickstart.md:102]:
```suggestion
```

File: docs/quickstart.md [hunk 29]
```diff
@@ -0,0 +1,157 @@
...
+
+    3. Enter your database credentials on the resulting page without changing any values.
```
Comment (catalinaadam) [inline docs/quickstart.md:104]:
```suggestion
    2.  On the **Add Service** page, enter your database credentials without changing any values.
```

File: docs/quickstart.md [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Optional: Enter the information on the *Labels* or *Additional Options* section.
```
Comment (catalinaadam) [inline docs/quickstart.md:106]:
```suggestion
    3. (Optional): Enter the information on the **Labels** or **Additional Options** section. 
```

File: docs/quickstart.md [hunk 31]
```diff
@@ -0,0 +1,157 @@
...
+
+    For detailed information, see [Adding a PostgreSQL database](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html).
+
+=== "Amazon RDS"
+
+    You can use PMM for monitoring [Amazon RDS](https://aws.amazon.com/rds/). By using the PMM web interface, you connect to the Amazon RDS DB instance. 
+
+    You only need to provide the [IAM user access key](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-access-key-for-an-iam-user) or assign an [IAM role](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-iam-role) and PMM discovers the Amazon RDS DB instances available for monitoring.
+
+    Before you add Amazon instance for moniitoring you should have the following:
```
Comment (catalinaadam) [inline docs/quickstart.md:118]:
```suggestion
   Prerequisites: Before you add an Amazon instance for monitoring, make sure to: ```

File: docs/quickstart.md [hunk 32]
```diff
@@ -0,0 +1,157 @@
...
+
+    - AWS RDS Access Key and RDS Secret Access Key. This key should have permission to monitor RDS.
```
Comment (catalinaadam) [inline docs/quickstart.md:120]:
```suggestion
    - Get your AWS RDS Access Key and RDS Secret Access Key. This key should have permission to monitor RDS.
```

File: docs/quickstart.md [hunk 33]
```diff
@@ -0,0 +1,157 @@
...
+    - Recommended: Enable *Enhanced Monitoring* option in the settings of your Amazon RDS DB instance.
+    - Database username and password with access to login to the RDS instance.
+    - Access to the RDS instance via a TCP port.
```
Comment (catalinaadam) [inline docs/quickstart.md:123]:
```suggestion
    - Access the RDS instance via a TCP port.
```

File: docs/quickstart.md [hunk 34]
```diff
@@ -0,0 +1,157 @@
...
+
+    You can use PMM for monitoring [Amazon RDS](https://aws.amazon.com/rds/). By using the PMM web interface, you connect to the Amazon RDS DB instance. 
+
+    You only need to provide the [IAM user access key](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-access-key-for-an-iam-user) or assign an [IAM role](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-iam-role) and PMM discovers the Amazon RDS DB instances available for monitoring.
+
+    Before you add Amazon instance for moniitoring you should have the following:
+
+    - AWS RDS Access Key and RDS Secret Access Key. This key should have permission to monitor RDS.
+    - Recommended: Enable *Enhanced Monitoring* option in the settings of your Amazon RDS DB instance.
+    - Database username and password with access to login to the RDS instance.
```
Comment (catalinaadam) [inline docs/quickstart.md:122]:
```suggestion
    - Set up your database username and password with access to login to the RDS instance.
```

File: docs/quickstart.md [hunk 35]
```diff
@@ -0,0 +1,157 @@
...
+    - Access to the RDS instance via a TCP port.
+
+    Here are the steps to add an Amazon RDS database instance for monitoring:
```
Comment (catalinaadam) [inline docs/quickstart.md:125]:
```suggestion
    To add an Amazon RDS database instance for monitoring:
```

File: docs/quickstart.md [hunk 36]
```diff
@@ -0,0 +1,157 @@
...
+
+    1. From the PMM UI, go to **Configuration** → **PMM Inventory** → **Add Instance**.
```
Comment (catalinaadam) [inline docs/quickstart.md:127]:
```suggestion
    1. From the PMM UI, go to **Configuration > PMM Inventory > Add Instance**.
```

File: docs/quickstart.md [hunk 37]
```diff
@@ -0,0 +1,157 @@
...
+
+    2. Select **Amazon RDS – Add a remote instance**.
+
+    3. Enter the **access key ID** and the **secret access key** of your IAM user or leave these fields empty if an IAM role was created.
```
Comment (catalinaadam) [inline docs/quickstart.md:131]:
```suggestion
    3. Enter the **access key ID** and the **secret access key** of your IAM user, or leave these fields empty if an IAM role was created.
```

File: docs/quickstart.md [hunk 38]
```diff
@@ -0,0 +1,157 @@
...
+
+    4. Click **Discover** for PMM to retrieve the available Amazon RDS instances.
+
+    5. For the instance that you would like to monitor, select **Start monitoring**.
+
+    6. Enter your database credentials on the resulting page without changing any values.
+
+    7. Optional: Enter the information on the **Labels** or **Additional Options** section.
```
Comment (catalinaadam) [inline docs/quickstart.md:139]:
```suggestion
    7. (Optional) Enter the information on the **Labels** or **Additional Options** section. 
```

File: docs/quickstart.md (hunk 13 above)
Comment (atymchuk) [reply docs/quickstart.md:61]: +1

File: docs/quickstart.md (hunk 17 above)
Comment (Nailya) [reply docs/quickstart.md:69]: why we mention "without changing any values". I think user should know that he/she can change any values.

File: docs/quickstart.md [hunk 39]
```diff
@@ -0,0 +1,158 @@
...
+
+    2. Select **Amazon RDS – Add a remote instance**.
+
+    3. Enter the **access key ID** and the **secret access key** of your IAM user, or leave these fields empty if an IAM role was created.
+
+    4. Click **Discover** for PMM to retrieve the available Amazon RDS instances.
+
+    5. For the instance that you would like to monitor, select **Start monitoring**.
+
+    6. Enter your database credentials on the resulting page without changing any values.
```
Comment (Nailya) [inline docs/quickstart.md:138]: same here, why user can't change any other values?

**Review by Nailya — CHANGES_REQUESTED** [review]

**Review by catalinaadam — COMMENTED** [review]
I have pushed my changes, probably Alina should review as well

File: mkdocs-base.yml [hunk 40]
```diff
@@ -153,53 +154,51 @@ nav:
...
+          - use/roles-and-permissions/access_roles.md
+          - use/roles-and-permissions/manage_roles.md
+          - use/roles-and-permissions/assign_roles.md
+          - use/roles-and-permissions/use_case.md
+        - how-to/upgrade.md
+        - how-to/troubleshoot.md
+        - how-to/secure.md
+        - how-to/optimize.md
   - Details:
       - details/index.md
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:200]: This page does not exist.

File: mkdocs-base.yml [hunk 41]
```diff
@@ -153,53 +154,51 @@ nav:
...
+      - details/interface.md
       - details/architecture.md
       - details/interface.md
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:203]: You have two "details/interface.md" pages in this section

File: mkdocs-base.yml [hunk 42]
```diff
@@ -78,7 +79,6 @@ markdown_extensions:
       emoji_generator: !!python/name:materialx.emoji.to_svg
 
 plugins:
-  search: {}
   section-index: {}
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:82]:
```suggestion
```

File: docs/quickstart.md [hunk 43]
```diff
@@ -0,0 +1,158 @@
...
+
+For PMM to work correctly, your system’s firewall should allow TCP traffic on these ports (UDP is not needed).
+
+## Install PMM
+
+!!! caution alert alert-warning "Important"
+    - You can use the [easy installation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/easy-install.html) script that will verify and install any missing software and dependencies. To use it, run the command with sudo privileges or as root.
+    - You can download the script from [github](https://github.com/percona/pmm/blob/main/get-pmm.sh).
+
+To install the PMM server, follows the steps below:
```
Comment (alina-derkach-oaza) [inline docs/quickstart.md:34]:
```suggestion
To install the PMM server, follow the steps below:
```

File: docs/quickstart.md [hunk 44]
```diff
@@ -0,0 +1,158 @@
...
+
+    4. (Optional) Enter the information on the **Labels** and **Additional Options** section. 
+
+    5. Click **Add Service** at the bottom.
+
+    For detailed information, see [Adding a MySQL database for monitoring](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html).
+
+=== "MongoDB"
+
+    Before you add a MongoDB database for monitoring, you should have a [database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html#create-pmm-account-and-set-permissions).
```
Comment (alina-derkach-oaza) [inline docs/quickstart.md:79]:
```suggestion
 **Prerequisites**
 Before adding a MongoDB database for monitoring,  [create a database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mongodb.html#create-pmm-account-and-set-permissions).
```

File: docs/quickstart.md [hunk 45]
```diff
@@ -0,0 +1,158 @@
...
+
+    For detailed information, see [Adding a PostgreSQL database](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html).
+
+=== "Amazon RDS"
+
+    You can use PMM for monitoring [Amazon RDS](https://aws.amazon.com/rds/). By using the PMM web interface, you connect to the Amazon RDS DB instance. 
+
+    You only need to provide the [IAM user access key](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-access-key-for-an-iam-user) or assign an [IAM role](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html#creating-an-iam-role) and PMM discovers the Amazon RDS DB instances available for monitoring.
+
+    Before you add Amazon instance for monitoring you should have the following:
```
Comment (alina-derkach-oaza) [inline docs/quickstart.md:119]:
```suggestion
    Before you add Amazon instance for monitoring, do the following:
```

File: docs/quickstart.md [hunk 46]
```diff
@@ -0,0 +1,158 @@
...
+    For detailed information, see [Adding an Amazon RDS instance for monitoring](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html).
+
+
+## Next steps
+
+Explore the following topics to gain a deeper understanding of PMM:
+
+- [Configure](how-to/configure.md) - Learn how to configure PMM via the interface.
+- [Manage users](how-to/manage-users.md) - Learn how to manage users in PMM.
+- [Roles and permissions](get-started/roles-and-permissions/index.md) - Learn more about roles and permissions in PMM.
```
Comment (alina-derkach-oaza) [inline docs/quickstart.md:153]: broken link

File: docs/quickstart.md [hunk 47]
```diff
@@ -0,0 +1,158 @@
...
+- [Backup and restore](get-started/backup/index.md) — Learn how to backup and to restore data in PMM.
```
Comment (alina-derkach-oaza) [inline docs/quickstart.md:154]: broken link

File: docs/quickstart.md [hunk 48]
```diff
@@ -0,0 +1,157 @@
...
+        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+        ```
+
+    === "wget"
+
+        ```sh
+        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+        ```
+
+3. Log in to PMM with the default login credentials that are provided after the installation is completed.
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/quickstart.md:53]: Done

File: docs/quickstart.md [hunk 49]
```diff
@@ -0,0 +1,157 @@
...
+
+For detailed instructions on installing the PMM server with the other methods, see [Setting up PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/index.html).
+
+## Add a database instance
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/quickstart.md:57]: @artemgavrilov I could not understand your comment properly. Could you please rephrase?

File: docs/quickstart.md [hunk 50]
```diff
@@ -0,0 +1,158 @@
+# Quickstart Guide
+
+
+This guide helps you quickly get started with PMM using Docker-compatible Linux-based systems.
+
+
+## Prerequisites
+
+[Docker](https://docs.docker.com/engine/install/) 1.12.6 or higher.
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/quickstart.md:9]: Done.

File: docs/quickstart.md [hunk 51]
```diff
@@ -0,0 +1,158 @@
...
+
+**Memory**
+
+Each database node should have at least 2 GB of memory for effective monitoring. The increase in memory usage is not proportional to the number of nodes. For example, data from 20 nodes should be easily handled with 16 GB.
+
+**Ports**
+
+Check out the [list of ports used by PMM components](setting-up/server/network.md).
+
+For PMM to work correctly, your system’s firewall should allow TCP traffic on these ports (UDP is not needed).
```
[the remark this answers is not in the document]
Comment (rasika-chivate) [reply docs/quickstart.md:26]: Done

File: docs/quickstart.md (hunk 39 above)
Comment (rasika-chivate) [reply docs/quickstart.md]: Done

File: mkdocs-base.yml (hunk 40 above)
Comment (rasika-chivate) [reply mkdocs-base.yml:199]: Its there.

File: docs/quickstart.md [hunk 52]
```diff
@@ -0,0 +1,165 @@
...
+    For detailed information, see [Adding an Amazon RDS instance for monitoring](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/aws.html).
+
+
+## Next steps
+
+Explore the following topics to gain a deeper understanding of PMM:
+
+- [Configure](how-to/configure.md) - Learn how to configure PMM via the interface.
+- [Manage users](how-to/manage-users.md) - Learn how to manage users in PMM.
+- [Roles and permissions](get-started/roles-and-permissions/index.md) - Learn more about roles and permissions in PMM.
```
Comment (catalinaadam) [inline docs/quickstart.md:160]:
```suggestion
- [Roles and permissions](use/roles-and-permissions/index.md) - Learn more about roles and permissions in PMM.
```

File: docs/quickstart.md [hunk 53]
```diff
@@ -0,0 +1,165 @@
...
+- [Backup and restore](get-started/backup/index.md) — Learn how to backup and to restore data in PMM.
```
Comment (catalinaadam) [inline docs/quickstart.md:161]:
```suggestion
- [Backup and restore](use/backup/index.md) — Learn how to backup and to restore data in PMM.
```

File: docs/quickstart.md (hunk 49 above)
[the remark this answers is not in the document]
Comment (BupycHuk) [reply docs/quickstart.md:60]: I think explanation should be more detailed.

**Review by catalinaadam — APPROVED** [review]

File: docs/details/interface.md [hunk 54]
```diff
@@ -1,3 +1,131 @@
...
+
+PMM's user interface is a browser application based on [Grafana].
+
+## Logging in
+
+1. Start a web browser and in the address bar enter the server name or IP address of the PMM server host.
+
+2. The page loads showing the PMM login screen.
+
+   ![!](../_images/PMM_Login.jpg)
```
Comment (alina-derkach-oaza) [inline docs/details/interface.md:13]:
```suggestion
    ![!](../_images/PMM_Login.jpg)
```

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: mkdocs-base.yml [hunk 55]
```diff
@@ -288,26 +298,26 @@ nav:
...
+      - "Manage allowed component versions": dbaas/get-started.html#manage-allowed-component-versions
+      - "Troubleshooting":
+        - "OLM installation": dbaas/troubleshoot-kubernetes.html#troubleshooting-olm-installation
+        - "Operators installation": dbaas/troubleshoot-kubernetes.html#troubleshooting-operators-installation
+    - "Managing Database clusters":
+      - "Add a DB Cluster": dbaas/databases.html#add-a-db-cluster
+      - "Manage a DB Cluster": dbaas/databases.html#edit-a-db-cluster
+      - "Delete a DB Cluster": dbaas/databases.html#delete-a-db-cluster
+      - "Create a database cluster from a template": dbaas/DBaaS_template.md
+    - "Backup and restore": dbaas/backup_restore.md
```
Comment (catalinaadam) [inline mkdocs-base.yml:320]:
```suggestion
    - "DBaaS Backup and restore": dbaas/backup_restore.md
```

Comment (catalinaadam) [reply mkdocs-base.yml:320]: we already have a B&R topic so we need to differentiate

File: docs/quickstart.md [hunk 56]
```diff
@@ -0,0 +1,165 @@
...
+- Approximately 1 GB of storage per monitored database node for data retention set to one week 
+
+**Memory**
+
+Each database node should have at least 2 GB of memory for effective monitoring. The increase in memory usage is not proportional to the number of nodes. For example, data from 20 nodes should be easily handled with 16 GB.
+
+**Ports**
+
+- By default, port 443 should be opened on the PMM Server.
+- The database port should be open for the PMM Server.
```
Comment (ademidoff) [inline docs/quickstart.md:23]:
```suggestion
- The database port should be open for the PMM Agent.
```

**Comment by rnovikovP** [comment]
https://github.com/percona/pmm-doc/pull/1159

**Pull request closed without merging**